### PR TITLE
Fix blockchain charts 30 days

### PR DESCRIPTION
**Pull request by Esteban100k**
With this PR, an initial load can be applied which allows me to capture the last 30 days, regardless of whether it is between 2 months, it will show me the days in the statistics

### CODE REVIEW

File: src/api/index.tsx [hunk 1]
```diff
@@ -395,11 +407,17 @@ export const fetchBlockchainChartUniqueAddresses = (dates: FilterDates) => {
 
 export const fetchBlockchainDailyGasUsed = (dates: FilterDates) => {
     return new Promise((resolve, reject) => {
+        let url = `${getBaseUrl()}${baseEndpoint}/dailyGasUsed?startTime=${
```
Comment (rafaelkordonias) [inline src/api/index.tsx:413]: you can use const

File: src/api/index.tsx [hunk 2]
```diff
@@ -395,11 +407,17 @@ export const fetchBlockchainChartUniqueAddresses = (dates: FilterDates) => {
...
+            dates.startDate
+        }&endTime=${dates.endDate}`
+
+        if (dates.limit > 0 && dates.limit !== undefined && dates.limit !== null) {
+            url = url + `&limit=${dates.limit}`
+        }
+
         let config = {
```
Comment (rafaelkordonias) [inline src/api/index.tsx:421]: you can use const

File: src/api/index.tsx [hunk 3]
```diff
@@ -423,11 +441,17 @@ export const fetchBlockchainDailyGasUsed = (dates: FilterDates) => {
 
 export const fetchBlockchainActiveAddresses = (dates: FilterDates) => {
     return new Promise((resolve, reject) => {
+        let url = `${getBaseUrl()}${baseEndpoint}/activeAddresses?startTime=${
```
Comment (rafaelkordonias) [inline src/api/index.tsx:447]: you can use const

File: src/api/index.tsx [hunk 4]
```diff
@@ -423,11 +441,17 @@ export const fetchBlockchainDailyGasUsed = (dates: FilterDates) => {
...
+            dates.startDate
+        }&endTime=${dates.endDate}`
+
+        if (dates.limit > 0 && dates.limit !== undefined && dates.limit !== null) {
+            url = url + `&limit=${dates.limit}`
+        }
+
         let config = {
```
Comment (rafaelkordonias) [inline src/api/index.tsx:455]: you can use const

File: src/api/index.tsx [hunk 5]
```diff
@@ -451,11 +475,17 @@ export const fetchBlockchainActiveAddresses = (dates: FilterDates) => {
 
 export const fetchBlockchainAverageBlockSize = (dates: FilterDates) => {
     return new Promise((resolve, reject) => {
+        let url = `${getBaseUrl()}${baseEndpoint}/averageBlockSize?startTime=${
```
Comment (rafaelkordonias) [inline src/api/index.tsx:481]: you can use const

File: src/api/index.tsx [hunk 6]
```diff
@@ -451,11 +475,17 @@ export const fetchBlockchainActiveAddresses = (dates: FilterDates) => {
...
+            dates.startDate
+        }&endTime=${dates.endDate}`
+
+        if (dates.limit > 0 && dates.limit !== undefined && dates.limit !== null) {
+            url = url + `&limit=${dates.limit}`
+        }
+
         let config = {
```
Comment (rafaelkordonias) [inline src/api/index.tsx:489]: you can use const

File: src/api/index.tsx [hunk 7]
```diff
@@ -473,11 +503,17 @@ export const fetchBlockchainAverageBlockSize = (dates: FilterDates) => {
 
 export const fetchBlockchainAverageGasPriceUsed = (dates: FilterDates) => {
     return new Promise((resolve, reject) => {
+        let url = `${getBaseUrl()}${baseEndpoint}/avgGasPriceUsed?startTime=${
```
Comment (rafaelkordonias) [inline src/api/index.tsx:509]: you can use const

File: src/api/index.tsx [hunk 8]
```diff
@@ -473,11 +503,17 @@ export const fetchBlockchainAverageBlockSize = (dates: FilterDates) => {
...
+            dates.startDate
+        }&endTime=${dates.endDate}`
+
+        if (dates.limit > 0 && dates.limit !== undefined && dates.limit !== null) {
+            url = url + `&limit=${dates.limit}`
+        }
+
         let config = {
```
Comment (rafaelkordonias) [inline src/api/index.tsx:517]: you can use const

File: src/api/index.tsx [hunk 9]
```diff
@@ -500,12 +536,18 @@ export const fetchBlockchainAverageGasPriceUsed = (dates: FilterDates) => {
 }
 
 export const fetchBlockchainDailyTokenTransfer = (dates: FilterDates) => {
+    let url = `${getBaseUrl()}${baseEndpoint}/dailyTokenTransfer?startTime=${
```
Comment (rafaelkordonias) [inline src/api/index.tsx:542]: you can use const

File: src/api/index.tsx [hunk 10]
```diff
@@ -500,12 +536,18 @@ export const fetchBlockchainAverageGasPriceUsed = (dates: FilterDates) => {
...
+        dates.startDate
+    }&endTime=${dates.endDate}`
+
+    if (dates.limit > 0 && dates.limit !== undefined && dates.limit !== null) {
+        url = url + `&limit=${dates.limit}`
+    }
+
     return new Promise((resolve, reject) => {
         let config = {
```
Comment (rafaelkordonias) [inline src/api/index.tsx:551]: you can use const

**Review by rafaelkordonias — CHANGES_REQUESTED** [review]

**Review by rafaelkordonias — APPROVED** [review]
LGTM

**Review by Ayagoumi — APPROVED** [review]

**Pull request merged**